tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -94,7 +98,11 @@ void start(String[] args) throws IOException {
                 record = new ProducerRecord<>(config.topicName, payload);
 
                 long sendStartMs = System.currentTimeMillis();
-                cb = new PerfCallback(sendStartMs, payload.length, stats);
+                if ( config.warmupRecords > 0 && i == config.warmupRecords ) {
+                    steadyStateStats = new Stats(config.numRecords - config.warmupRecords, config.warmupRecords > 0);
+                    stats.steadyStateActive = true;
```

Comment:
It seems this is used to stop the `printWindow`, right? If so, maybe the better naming is `stopPrint`. WDYT?